src/consumer/assigners/roundRobinAssigner/index.js

```diff
@@ -41,6 +41,9 @@ module.exports = ({ cluster }) => ({
       assignment[memberId] = {}
     })
 
+    // Make sure the assigner always work with up to date metadata
+    await cluster.refreshMetadataIfNecessary()
```

Comment:
I think we should move this to the consumerGroup before calling the assigner, so that we don't need to add this to every assigner.

Reply:
fe54b58